clients/cobol-lsp-vscode-extension/src/services/ProcessorGroups.ts

```diff
@@ -263,7 +287,7 @@ async function loadProcessorGroupSettings<T extends string | string[]>(
     return configObject;
   }
   try {
-    if (dialect && dialect !== "COBOL") {
+    if (dialect && dialect !== "COBOL" && PREPROCESSOR in pgCfg) {
```

Comment:
Is this the only use of `PREPROCESSOR`?
```suggestion
    if (dialect && dialect !== "COBOL" && "preprocessor" in pgCfg) {
```